frontend/src/Response/SimpleQuestions/SimpleQuestion.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box } from "@mui/material";
+import CheckboxQuestion, { CheckboxQuestionInterface } from "./CheckboxQuestion";
+import InputInfo from "./InputInfo";
+import NumberQuestion, { NumberQuestionInterface } from "./NumberQuestion";
+import SelectQuestion, { SelectQuestionInterface } from "./SelectQuestion";
+import TextQuestion, { TextQuestionInterface } from "./TextQuestion";
+
+export enum SimpleQuestionType {
+    TEXT = "TEXT",
+    NUMBER = "NUMBER",
+    SELECT = "SELECT",
+    CHECKBOX = "CHECKBOX",
+}
+
+export type SimpleQuestionTypesList = (
+    TextQuestionInterface |
+    NumberQuestionInterface |
+    SelectQuestionInterface |
+    CheckboxQuestionInterface
+);
+
+export interface SimpleQuestionInterface {
+    questionType: SimpleQuestionType;
+    questionData: SimpleQuestionTypesList;
+}
+
+
+function SimpleQuestion(questionData: SimpleQuestionInterface): JSX.Element {
+    let component;
```

Comment:
It's better to write just `return (<Component ... > ... </Component>)` in each case instead of using `component` variable.
And it's not cool that `component` variable is untyped